typings/vscode-mssql.d.ts

```diff
@@ -178,6 +178,22 @@ declare module 'vscode-mssql' {
 		osVersion: string;
 	}
 
+	/**
+	 * The possible values of the server engine edition
+	 */
+	export const enum DatabaseEngineEdition {
+		Unknown = 0,
+		Personal = 1,
+		Standard = 2,
+		Enterprise = 3,
+		Express = 4,
+		SqlDatabase = 5,
+		SqlDataWarehouse = 6,
+		SqlStretchDatabase = 7,
+		SqlManagedInstance = 8,
+		SqlOnDemand = 11
```

Comment:
And this to `SynapseServerlessSqlPool`

Reply:
Raised a new issue https://github.com/microsoft/azuredatastudio/issues/20617 to keep track for discussion.